docs/reference/library/math.md

```diff
@@ -86,15 +86,75 @@ $ (3x + y) / 7 &= 9 && "given" \
 ```
 
 # Math fonts
-You can set the math font by with a [show-set rule]($styling/#show-rules) as
-demonstrated below. Note that only special OpenType math fonts are suitable for
-typesetting maths.
+The default math font is `New Computer Modern Math`. As demonstrated below, you
+can tweak it with [show-set rules]($styling/#show-rules). The rule's selector
+can be the general `{math.equation}`, specific [symbols](#symbols) and texts, or
+[math functions](#function-calls) like `{math.op}`.
 
-```example
+```example:"Change the overall math font"
 #show math.equation: set text(font: "Fira Math")
 $ sum_(i in NN) 1 + i $
 ```
 
+```example:"Change the font for a specific character"
+#show math.equation: it => {
+  show "{": set text(font: "STIX Two Math", fill: maroon)
+  it
+}
+$ f(x, y) := cases(0 "if" x < 0, x "otherwise") $
+```
+
+As in the regular text layout, the [`font`]($text.font) parameter also accepts
+a priority list of font family descriptor. In the example below, the font
+`Noto Sans Math` covers capital serif italic letters, and the default font
+`New Computer Modern Math` covers the others. The letters `𝐴` and `𝑍` in the
+[regex] are [mathematical alphanumeric symbols](https://en.wikipedia.org/wiki/Mathematical_Alphanumeric_Symbols)
+defined in the Unicode standard, instead of the regular `A` and `Z` in ASCII.
+Besides, some characters do not belong to this Unicode block for historic
+reasons, making it harder to match other ranges. For example, `{regex("[𝑎-𝑧]")}`
+does not match `ℎ` and the dotless `𝚤` and `𝚥`.
+
+```example:"Change the font for a range of characters"
+#show math.equation: set text(font: (
+  (name: "Noto Sans Math", covers: regex("[𝐴-𝑍]")),
+  "New Computer Modern Math",
+))
+$ 2A + B = C. $
+```
+
+In addition to [`font`]($text.font), the rule can also change OpenType
+[features]($text.features) including [stylistic sets]($text.stylistic-set) and
+character variants.
+
+```example:"Configure OpenType features"
+#show math.equation: set text(
+  // Prefer upright integrals (ss02) and small capitals (ss05)
+  stylistic-set: (2, 5),
+  // Use a slashed circle for ∅, replacing the default slashed zero
+  features: ("cv01",),
+)
+// These features are defined by New Computer Modern Math.
+// Other fonts may not support them or map them to different features.
+
+$ integral f dif x, a inter bb(N) = nothing. $
+```
+
+Note that typesetting maths involves complex positioning and spacing, requiring
+a specially designed [OpenType math font](https://learn.microsoft.com/typography/opentype/spec/math).
+Typst will take the first font without any [`covers`]($text.font) as the base
+font, and extract typographic metrics for maths from it. If you have to use
+non-math fonts for certain glyphs, specify a coverage to tell Typst select the
+base from other fonts.
+
+```example
+#show math.equation: set text(font: (
+  (name: "New Computer Modern Math", covers: "latin-in-cjk"),
```

Comment:
`"latin-in-cjk"` has to be replaced with `regex("\p{Latin}")`, if the first font does not support math.
Otherwise, the `{` of `cases` won't be stretched.
<img width="1767" height="432" alt="图片" src="https://github.com/user-attachments/assets/cc56c8ee-b42e-435e-a83d-69d0978ba687" />
<img width="1771" height="412" alt="图片" src="https://github.com/user-attachments/assets/3be7c6f6-7530-48c7-a5ed-2aed88de1323" />